README.md

```diff
@@ -1,19 +1,218 @@
 # **Allstar**
 
-GitHub App installed on organizations or repositories to set and enforce security policies.
+Allstar is a GitHub App installed on organizations or repositories to set and
+enforce security policies. It's goal is to be able to detect any GitHub setting,
+or repository file contents that may be risky or not follow security best
+practices. Then, if Allstar finds a repository to be out of compliance, it will
```

Comment:
```suggestion
enforce security policies. Its goal is to be able to detect any GitHub setting
or repository file contents that may be risky or not follow security best
practices. If Allstar finds a repository to be out of compliance, it will
```